server/rpm/pbench-server.spec.j2

```diff
@@ -9,16 +9,16 @@ Source0:        pbench-server-%{version}.tar.gz
 Buildarch:      noarch
 
 
-Requires: python3
+Requires: python3 python3-devel gcc
```

Comment:
This was required for compiling psutil, so in the end it may not be needed on the server, depending on how things shake out with the requirements files, but better safe than sorry.